### PR TITLE
Add Feedback Link

### LINKED ISSUE #742: Add link to each page for feedback
"Have feedback, click here"

This will link the user to a GH issue template in this repo

### PR DESCRIPTION
**Pull request by jesse-martinez**
Fixes #742

### CODE REVIEW

**Comment by jesse-martinez** [comment]
@bryanfriedman @BrianMMcClain Let's go ahead and merge this. Then we can focus on adding the feedback link to workshops.

File: themes/docsy/layouts/videos/single.html [hunk 1]
```diff
@@ -23,6 +23,13 @@ <h2 class="mb-4"> {{- .Title -}}</h2>
             <p>{{ .Description }}</p>
         </div>
     </div>
+    <div class="row justify-content-center mb-5 text-center">
+        <div class="col-lg-9 col-12 px-0 d-flex">
+          <div class="feedback mt-4 pt-4 w-100 text-left border-top">
+            <a href='{{ printf "%s%s" "https://github.com/vmware-tanzu/tanzu-dev-portal/issues/new?assignees=&labels=bug&template=bug_report.md&title=" .Title }}'><i class="fab fa-github mr-2"></i><span class="align-top">Report an issue</span></a>
```
Comment (BrianMMcClain) [inline themes/docsy/layouts/videos/single.html:29]:
```suggestion
            <a href='{{ printf "%s%s" "https://github.com/vmware-tanzu/tanzu-dev-portal/issues/new?assignees=&labels=bug&template=bug_report.md&title=" .Title }}'><i class="fab fa-github mr-2"></i><span class="align-top">Report an issue</span></a>
```

I think we might have to do this with each link, but we might have to do some explicit URL encoding on this. I thought it was automatic, but for example, [this video](https://deploy-preview-937--tanzu-dev-portal.netlify.app/developer/videos/spring-cloud-data-flow-creating-and-deploying-streams/) creates an issue with the title of just "Creating"

**Comment by BrianMMcClain** [comment]
@jesse-martinez Just the one thing I ran into, otherwise looks great! Instead of `printf`'ing the whole URL, it might actually work just by having the HTML, and then doing something along the lines of `...&title={{ .Title }}` like it is in this [mockup for feedback on workshops](https://github.com/vmware-tanzu/tanzu-dev-portal/pull/918/files#diff-3575ba7dcdf5c1e58d5b3b098e22f817b3faca604c94d903350c407d758580e9R69).

File: themes/docsy/layouts/videos/single.html (hunk 1 above)
Comment (jesse-martinez) [reply themes/docsy/layouts/videos/single.html]: Hey @BrianMMcClain , I swapped out the printf method with your method and it looks to resolve this issue as well. Ready to merge?

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**